### DISCUSSION TITLE
amamba/user-guide/pipeline/jenkins-agent/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#4990] [Announcements]
# amamba/user-guide/pipeline/jenkins-agent/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/amamba/user-guide/pipeline/jenkins-agent/

**Comment by san3Xian** [comment]
注意自定义agent 中定义的containers，不会默认继承原本template中的volumeMounts定义
注意自定义agent 产生的流水线pod，也不会默认继承原本template中的yaml 声明下的affinity定义

**Comment by gssjl2008** [comment]
global-jenkins-casc-config 是  jenkins-casc-config  吧？
